t/cobrand/zurich.t

```diff
@@ -515,18 +521,28 @@ subtest "external report triggers email" => sub {
     FixMyStreet::override_config {
         ALLOWED_COBRANDS => [ 'zurich' ],
     }, sub {
+
+        # required to see body_external field
+        $report->state('planned');
+        $report->set_extra_metadata('closure_status' => 'closed');
         # Set the public_response manually here because the default one will have line breaks that get escaped as HTML, causing the comparison to fail.
         $report->set_extra_metadata('public_response' => 'Freundliche Gruesse Ihre Stadt Zuerich');
-        $report->update({ state => 'closed' }); # required to see body_external field
+        $report->update;
+
         $mech->get_ok( '/admin/report_edit/' . $report->id );
+        $mech->form_with_fields( 'publish_response' );
         $mech->submit_form_ok( {
+            button => 'publish_response',
             with_fields => {
                 body_external => $external_body->id,
                 external_message => $EXTERNAL_MESSAGE,
             } });
+        $report->discard_changes;
         $mech->get_ok( '/report/' . $report->id );
     };
-    $mech->content_contains('Extern');
+    is ($report->state, 'closed', 'Report was closed correctly') or return; # skipping remaining tests
```

Comment:
Not sure why the remaining tests would want to be skipped here instead of allowing the test to file, so `or return` should probably be removed.